contrib/babelfishpg_tsql/src/pl_funcs-2.c

```diff
@@ -629,13 +630,29 @@ dump_stmt_usedb(PLtsql_stmt_usedb *stmt_usedb)
 	printf("USE %s\n", stmt_usedb->db_name);
 }
 
+void get_grantees_names(List *grantees, StringInfo grantees_names)
+{	
+	for(int i = 0; i < list_length(grantees); i++)
+	{
+		char *grantee_name = (char *) list_nth(grantees, i);
+		if(i < list_length(grantees) - 1)
+			appendStringInfo(grantees_names, "%s, ", grantee_name);
+		else
+			appendStringInfo(grantees_names, "%s", grantee_name);
+	}
+}
+
```

Comment:
please explain this diff

Reply:
This part is suggested and reviewed by Shalini, if you want we can discuss that.